oci/private/structure_test.bzl

```diff
@@ -0,0 +1,62 @@
+"""Loading oci_tarball into the daemon and executes structure_test executable
+
+For some reason if DOCKER_HOST needs to be specified, use --test_env=DOCKER_HOST='unix://<path_to_sock>'
+Beware that this is going to lead to cache misses.
+"""
+attrs = {
+    "image": attr.label(mandatory = True, allow_single_file = True, doc = "Label to an oci_image target"),
+    "config": attr.label_list(allow_files = True, mandatory = True),
+}
+
+CMD = """\
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
+
+readonly ST_PATH="{st_path}"
+readonly FIXED_ARGS="{fixed_args}"
+
+"$ST_PATH" test $FIXED_ARGS $@
```

Comment:
```suggestion
#!/usr/bin/env bash

exec "{st_path}" test {fixed_args} "$@"
```

Reply:
I am not sure if we are consistent about invoking the tools. this differs from what we've recently merged. https://github.com/bazel-contrib/rules_oci/blob/5b41558ede238413d9ae6025f7021a106da95241/oci/private/container.sh.tpl#L39-L42

Reply:
You are comparing apples and oranges. One is a full script with different code paths, the other is a wrapper around a single tool and some args. https://tldp.org/LDP/abs/html/x17974.html